contracts/arbitration/GTCR.sol

```diff
@@ -0,0 +1,1013 @@
+/* solium-disable max-len*/
+
+pragma solidity ^0.5.10;
+
+import "./Arbitrable.sol";
+import "./CappedMath.sol";
+
+/**
+ *  @title GeneralizedTCR
+ *  This contract is a curated registry for any types of items. Just like TCR contract it uses request-challenge protocol and crowdfunding, but also has new features such as badges and request cancellation.
+ */
+contract GeneralizedTCR is Arbitrable {
```

Comment:
Once imported, this contract should implement IArbitrable, instead of extending deprecated `Arbitrable` contract